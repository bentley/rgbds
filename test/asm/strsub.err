warning: strsub.asm(15) -> strsub.asm::xstrsub(4): [-Wbuiltin-args]
    STRSUB: Length too big: 32
warning: strsub.asm(16) -> strsub.asm::xstrsub(4): [-Wbuiltin-args]
    STRSUB: Length too big: 300
warning: strsub.asm(17) -> strsub.asm::xstrsub(4): [-Wbuiltin-args]
    STRSUB: Position starts at 1
warning: strsub.asm(17) -> strsub.asm::xstrsub(4): [-Wbuiltin-args]
    STRSUB: Length too big: 300
warning: strsub.asm(19) -> strsub.asm::xstrsub(4): [-Wbuiltin-args]
    STRSUB: Position 4 is past the end of the string
warning: strsub.asm(19) -> strsub.asm::xstrsub(4): [-Wbuiltin-args]
    STRSUB: Length too big: 1
warning: strsub.asm(22) -> strsub.asm::xstrsub(4): [-Wbuiltin-args]
    STRSUB: Length too big: 10
