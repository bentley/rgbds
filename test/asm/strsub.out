A
B
C
A
B
C
AB
BC
BC
BC
ABC


カタ
カナ
カナ
g
g̈
